lib/router/app_router.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:auto_route/auto_route.dart';
+import 'app_router.gr.dart';
+
+
```

Comment:
Remove additional spacing. 1 line of space is enough between imports and rest of the code